### ISSUE TITLE
Can I read baseURL from Nuxt .env?

### ISSUE THREAD

**Issue by PrimozRome** [robsontenorio/vue-api-query#37]
Do I really have to explicitly define base URL again in the base model definition?

My nuxt.js configuration already sets axios parameters like base url etc.

Thankx

**Comment by robsontenorio** [comment]
Real world usage

`.env` file placed on root folder
```
API_URL =  http://my-api.com
```

nuxt.config.js
```js

// this package is provided by @nuxtjs/dotenv module
require('dotenv').config()

module.exports = {

  // this comes from .env file
  env: {
    API_URL: process.env.API_URL
  },

  // ... another configs ...

```

Your base model
```js
import { Model as BaseModel } from 'vue-api-query'

export default class Model extends BaseModel {
  // This comes from nuxt.config.js
  baseURL () {
    return process.env.API_URL
  }

  // implement a default request method
  request (config) {
    return this.$http.request(config)
  }
}
```

**Comment by PrimozRome** [comment]
Yeah, I didnt know process is accessible from the Model file as well. Thanks I will try.

**Comment by derekphilipau** [comment]
In real-world use I think most people will not be hard-coding the base URL into a class definition.  Hard-coding makes it impossible to test and deploy.  So I think that use of an .env environment variable should be the default use-case, best practice, or at least _also_ listed in the documentation after the hard-coded version.

**Comment by robsontenorio** [comment]
With Nuxt just define a process.env.API_URL and it will be used as baseUrl for axios calls. See Nuxt community dotenv-module

**Comment by derekphilipau** [comment]
Yes, I know, my point is that use of .env should be the _default_ usage in the documentation.  I can't ever imagine a good use-case for directly hard-coding a URL in the class file.

**Comment by njxqlus** [comment]
Ok, let's see to my problem. I use nuxt community dot env module. I set up `API_URL` in my .env file. I repeat in `nuxt.config.js` `env:{API_URL: process.env.API_URL}`.

I set up in `Model.js`: `baseURL () {return process.env.API_URL}`.

But in only works with normal requests (e.g. `SomeModel.get()`). It's not works with `SomeModel.custom({id: SomeModel.id}'/example')`. But when I export env variable (`export API_URL=my_api_url`) in works perfectly.

What's wrong?
